lib/block_view.go

```diff
@@ -543,9 +558,12 @@ type PostEntry struct {
 	// How many comments this post has
 	CommentCount uint64
 
-	// Indicator if a post is pinned or not.
+	// Indicator if a post is pinned or not to the user's profile.
 	IsPinned bool
 
+	// Indicator if a post is pinned to the global feed.
+	IsGlobalPinned bool
```

Comment:
Hmm.. I don't think we want anyone to have the ability to pin to the global feed do we?

Reply:
This is just the admin ability to pin to global feed that already exists. It was renamed in this PR.